docs/distributed-tracing.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[distributed-tracing]]
+== Distributed tracing
+
+// Content from APM Overview documentation
+// https://www.elastic.co/guide/en/apm/get-started/current/distributed-tracing.html
+Distributed tracing enables you to analyze performance throughout your microservices architecture all in one view.
+This is accomplished by tracing all of the requests --
+from the initial web request to your front-end service -- to queries made to your back-end services.
+This makes finding possible bottlenecks throughout your application much easier and faster.
+
+Elastic APM automatically supports distributed tracing,
+but there are some cases, outlined below, where additional setup is necessary.
+
+[float]
+[[tracing-rum-correlation]]
+=== Real User Monitoring (RUM) correlation
+
+// Content from RUM documentation
+// https://www.elastic.co/guide/en/apm/agent/rum-js/master/distributed-tracing-guide.html
+If your backend service generates an HTML page dynamically,
+the trace ID and parent span ID must be injected into the page when the RUM Agent is initialized.
+This ensures that the web browser’s page load appears as the root of the trace,
+and allows you to analyze the time spent in the browser vs in backend services.
+
+// Content from transaction-api.asciidoc
+include::./transaction-api.asciidoc[tag=ensure-parent-id-snippet]
+
+For more information, see <<transaction-ensure-parent-id>>.
+
+[float]
+[[tracing-custom-protocol]]
+=== Custom protocols
+
+Distributed tracing is automatically supported with HTTP/HTTPS.
+If you're using another protocol, like TCP, UDP, WebSocket, or Protocol Buffers,
+there are a few manual setup steps you must follow.
+
+In a distributed trace, multiple transactions are linked together with a `traceparent`.
+To create your own distributed trace, you must pass the current `traceparent` from an outgoing service,
+to a receiving service, and create a new transaction as a child of that `traceparent`:
+
+1. In one service, start a transaction with <<apm-start-transaction>>, or a span with <<apm-start-span>>.
```

Comment:
If not specifying a manual link-text, it will use the headline from the linked document, which is quite verbose in my opinion:

![image](https://user-images.githubusercontent.com/10602/75320659-75d92280-586f-11ea-8f56-c3486d36286d.png)

I think it would look better and be less confusing if the linked text was just `apm.startTransaction()` and `apm.startSpan()`. Then the reader can click the link to see which arguments they can provide.

The same goes for a few of the other links below.